invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/index.js

```diff
@@ -43,7 +43,7 @@ const TAB_PANES = [
 ];
 
 const replaceURLPathname = (newPathname) =>
-  window.history.replaceState({}, "", newPathname);
+  window.history.replaceState({}, "", `/me/${newPathname}`);
```

Comment:
thanks for fixing this, I missed it
But this is the reason why we need centralised links, because it is easily missed when they are scattered around